apps/tlon-mobile/src/screens/ChannelScreen.tsx

```diff
@@ -30,6 +34,23 @@ export default function ChannelScreen(props: ChannelScreenProps) {
   const { result: contacts } = db.useContacts();
 
   const { top, bottom } = useSafeAreaInsets();
+  const { ship } = useShip();
+
+  const messageSender = async (content: JSONContent, channelId: string) => {
+    if (!ship) {
+      return;
+    }
+
+    const isDm = whomIsDm(channelId);
```

Comment:
can we pull this off `channel.type`? More broadly, would be nice to constrain backend terminology like `whom` to `shared/api`

Reply:
Yeah, good idea.